ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_hbac_functional.py
         template: *ci-master-latest
-        timeout: 3600
-        topology: *master_1repl_1client
+        timeout: 8600
```

Comment:
In the temp commit, the timeout is set to 8600 but in the other prci_definition files the value is set to 7200. Which value is actually needed?

I would also add the test to nightly_latest_sssd.yaml as the rules are enforced by SSSD client.